frontend-react/src/pages/About.tsx

```diff
@@ -18,14 +21,20 @@ export const About = () => {
                     labs, and more through a single connection.
                 </h2>
                 <p>
-                    <button className="usa-button usa-button--outline">
-                        <USLink href="/product">
-                            Learn more about ReportStream{" "}
-                        </USLink>
-                    </button>
-                    <button className="usa-button usa-button--outline">
-                        <USLink href="/support/contact">Contact us</USLink>
-                    </button>
+                    <Button
```

Comment:
Woohoo!  Thanks for fixing this as well!

Reply:
Tried to catch all those nasty links-in-buttons :D